app/helpers/dossier_helper.rb

```diff
@@ -29,8 +29,8 @@ def url_for_new_dossier(revision)
     new_dossier_url(procedure_id: revision.procedure.id, brouillon: revision.draft? ? true : nil)
   end
 
-  def autosave_available?(dossier)
-    dossier.brouillon?
+  def autosave_available?(champ)
+    (champ.private? || champ.dossier.brouillon?) && !champ.repetition?
```

Comment:
on tente des patch sur la preview
```suggestion
    (champ.private? || champ.dossier.brouillon?) && !champ.repetition? && !champ.dossier.for_procedure_preview?
```